themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -125,8 +124,121 @@ function checkItemStatus(el) {
   var id = $item.find('.hiddenId').val();
   itemQueueAjax(id + '', $item);
 }
-
+*/
 var itemStatusObserver = null;
+
+var ItemStatusHandler = {
```

Comment:
It looks like you've done some significant reorganization of the item status handling code. Might this be worth separating out into its own pull request, or is it too tangled up with overdrive-specific logic? (I haven't read the code closely enough to tell -- I just know that one of our goals is to contain more of our JS code in objects for better control and clarity, and it seems like that's what you've done here).

Reply:
I can if you prefer.  Most of the changes are to make the code more extendable and then a few lines that actually extend the base object for getting statuses for the overdrive integration.  But either way, it should continue to work as before without Overdrive.

Reply:
If you don't mind, that would be great! When I find stand-alone chunks in a pull request, I like to separate them out as it makes the overall review process a bit easier, and also lets us gain some benefits to the code that much more quickly.